frontend/src/libs/import-export/undefinedToNull.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Util functions to turn undefined to null for json
+ *
+ * @exports
+ * @param {Object} item
+ * @returns{Object}
+ */
+export function jsonUndefinedToNull(items) {
+    const newItems = items.map(function (item) {
+        for (const key of Object.keys(item)) {
+            item[key] = item[key] ?? null;
+        }
+        return item;
+    });
+    return newItems;
+}
+
+/**
+ * Util functions to turn undefined to null for spreadsheets
+ *
+ * @exports
+ * @param {Object} item
+ * @returns{Object}
+ */
+export function spreadsheetUndefinedToNull(items) {
+    const newItems = items.map(function (item) {
```

Comment:
It is more concise to use arrow functions inside array functions like map, some and forEach.

Reply:
Changed to arrow functions.